api/src/main/java/com/mtech/vmcs/model/entity/InitialFileData.java

```diff
@@ -0,0 +1,23 @@
+package com.mtech.vmcs.model.entity;
+
+import lombok.Getter;
+import lombok.Setter;
+import java.util.List;
+
+public class InitialFileData {
+  @Getter
```

Comment:
You can add them on class, please refer to other entity.

Reply:
done, plz review it again